internal/performance/api_load.go

```diff
@@ -0,0 +1,162 @@
+package performance
+
+import (
+	"io/ioutil"
+	"encoding/base64"
+	"github.com/smartcontractkit/chainlink/internal/cltest"
+	"github.com/tsenart/vegeta/lib"
+	"fmt"
+	"time"
+	"log"
+	"github.com/smartcontractkit/chainlink/store"
+	"net/http"
+)
+
+var (
+	schemaPrefix = "../../internal/fixtures/web/"
+	schemas      = [...]string{
+		"hello_world_job.json",
+		"uint256_job.json",
+	}
+)
+
+// Pre-load the schemas prior to run-time to reduce overhead
+func GetSchemas() (schemaBytes [][]byte) {
+	for _, schema := range schemas {
+		byteArray, err := ioutil.ReadFile(schemaPrefix + schema)
+		if err != nil {
+			log.Fatal(err)
+		}
+		schemaBytes = append(schemaBytes, byteArray)
+	}
+	return schemaBytes
+}
+
+// Get the basic auth http header
+func GetBasicAuthHeader(store *store.Store) (header *http.Header) {
+	header = &http.Header{}
+	authString := store.Config.BasicAuthUsername + ":" + store.Config.BasicAuthPassword
+	header.Add("Authorization", "Basic "+base64.StdEncoding.EncodeToString([]byte(authString)))
+	return
+}
+
+// Create the vegata targets based on the schemas used for creating jobs
+func GetPostJobTargets(app *cltest.TestApplication) (targets []vegeta.Target) {
+	schemas := GetSchemas()
+	for _, schema := range schemas {
+		targets = append(targets, vegeta.Target{
+			Method: "POST",
+			URL:    fmt.Sprintf("%s/v2/jobs", app.Server.URL),
+			Body:   schema,
+			Header: *GetBasicAuthHeader(app.Store),
```

Comment:
In `GetBasicAuthHeader`, you're instantiating a struct and taking it's pointer, then returning a pointer, and taking its value here. Why not just return a value from `GetBasicAuthHeader`?